openmetadata-ui/src/main/resources/ui/src/pages/EntityImport/BulkEntityImportPage/BulkEntityImportPage.tsx

```diff
@@ -0,0 +1,602 @@
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+import ReactDataGrid from '@inovua/reactdatagrid-community';
+import '@inovua/reactdatagrid-community/index.css';
+import {
+  TypeColumn,
+  TypeComputedProps,
+} from '@inovua/reactdatagrid-community/types';
+import { Button, Card, Col, Row, Space, Typography } from 'antd';
+import { AxiosError } from 'axios';
+import React, {
+  MutableRefObject,
+  useCallback,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from 'react';
+import { useTranslation } from 'react-i18next';
+import { usePapaParse } from 'react-papaparse';
+import { useHistory, useParams } from 'react-router-dom';
+import {
+  CSVImportAsyncWebsocketResponse,
+  CSVImportJobType,
+} from '../../../components/BulkImport/BulkEntityImport.interface';
+import Banner from '../../../components/common/Banner/Banner';
+import { ImportStatus } from '../../../components/common/EntityImport/ImportStatus/ImportStatus.component';
+import TitleBreadcrumb from '../../../components/common/TitleBreadcrumb/TitleBreadcrumb.component';
+import { TitleBreadcrumbProps } from '../../../components/common/TitleBreadcrumb/TitleBreadcrumb.interface';
+import PageLayoutV1 from '../../../components/PageLayoutV1/PageLayoutV1';
+import Stepper from '../../../components/Settings/Services/Ingestion/IngestionStepper/IngestionStepper.component';
+import { UploadFile } from '../../../components/UploadFile/UploadFile';
+import {
+  ENTITY_IMPORT_STEPS,
+  VALIDATION_STEP,
+} from '../../../constants/BulkImport.constant';
+import { SOCKET_EVENTS } from '../../../constants/constants';
+import { useWebSocketConnector } from '../../../context/WebSocketProvider/WebSocketProvider';
+import { EntityType } from '../../../enums/entity.enum';
+import { CSVImportResult } from '../../../generated/type/csvImportResult';
+import { useFqn } from '../../../hooks/useFqn';
+import {
+  importEntityInCSVFormat,
+  importServiceInCSVFormat,
+} from '../../../rest/importExportAPI';
+import {
+  getCSVStringFromColumnsAndDataSource,
+  getEntityColumnsAndDataSourceFromCSV,
+} from '../../../utils/CSV/CSV.utils';
+import entityUtilClassBase from '../../../utils/EntityUtilClassBase';
+import Fqn from '../../../utils/Fqn';
+import { showErrorToast, showSuccessToast } from '../../../utils/ToastUtils';
+import './bulk-entity-import-page.less';
+
+let inEdit = false;
+
+const BulkEntityImportPage = () => {
+  const { socket } = useWebSocketConnector();
+  const [activeAsyncImportJob, setActiveAsyncImportJob] =
+    useState<CSVImportJobType>();
+  const activeAsyncImportJobRef = useRef<CSVImportJobType>();
+
+  const [activeStep, setActiveStep] = useState<VALIDATION_STEP>(
+    VALIDATION_STEP.UPLOAD
+  );
+  const activeStepRef = useRef<VALIDATION_STEP>(VALIDATION_STEP.UPLOAD);
+
+  const { t } = useTranslation();
+  const { entityType } = useParams<{ entityType: EntityType }>();
```

Comment:
Can we add unit tests for this as well?

Reply:
Will be modifying this file for the Glossary import centralization. Will add the unit test in that one!